docs_user/assemblies/assembly_configuring-isolated-networks.adoc

```diff
@@ -6,29 +6,18 @@ ifdef::context[:parent-context: {context}]
 
 = Configuring isolated networks
 
-At this point, you should have a good idea about VLAN and IPAM configuration
-you would like to replicate in the new environment.
+Before you begin replicating your existing VLAN and IPAM configuration in the {rhos_long} environment, you must have the following IP address allocations for the new control plane services:
 
-Before proceeding, you should have a list of the following IP address
-allocations to be used for the new control plane services:
-
-* 1 IP address, per isolated network, per {rhocp_long} worker node. (These
-  addresses configure openshift worker nodes to
-  `NodeNetworkConfigurationPolicy` custom resources (CRs).) For more information, see xref:configuring-openshift-worker-nodes_isolated-networks[Configuring {rhocp_long} worker nodes].
-* IP range, per isolated network, for the data plane nodes. (These ranges will
-  configure data plane nodes to `NetConfig` CRs.) For more information, see xref:configuring-data-plane-nodes_isolated-networks[Configuring data plane nodes].
-* IP range, per isolated network, for control plane services. (These ranges
-  will enable pod connectivity to isolated networks to
-  `NetworkAttachmentDefinition` CRs.) For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
-* IP range, per isolated network, for load balancer IP addresses. (These ranges will define load balancer IP addresses to `IPAddressPool` CRs for MetalLB.) For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
-
-[IMPORTANT]
-Make sure you have the information listed above before proceeding with the next steps.
+* 1 IP address, per isolated network, per {rhocp_long} worker node. These IP addresses configure {OpenShiftShort} worker nodes to the `NodeNetworkConfigurationPolicy` custom resources (CRs). For more information, see xref:configuring-openshift-worker-nodes_isolated-networks[Configuring {OpenShiftShort} worker nodes].
+* 1 IP range, per isolated network, for the data plane nodes. These ranges
+configure data plane nodes to `NetConfig` CRs. For more information, see xref:configuring-data-plane-nodes_isolated-networks[Configuring data plane nodes].
+* 1 IP range, per isolated network, for control plane services. These ranges
+enable pod connectivity to isolated networks to
+`NetworkAttachmentDefinition` CRs. For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
+* 1 IP range, per isolated network, for load balancer IP addresses. These IP ranges define load balancer IP addresses for MetalLB in the `IPAddressPool` CRs. For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
```

Comment:
I have a couple of questions on the bulleted list:

1. Is it correct that they need 1 IP range per isolated network for data plane nodes, control plane services, and load balancer IP addresses?
2.  Do the IP addresses really "configure" the worker nodes to the custom resource? Or do you mean to say “You configure these IP addresses in the `NodeNetworkConfigurationPolicy` custom resources (CRs) for the OpenShift worker nodes”? Similar question for the following bullet points, e.g., should the second bullet point say "You configure IP ranges in the `NetConfig` CRs for the data plane nodes"?